Workout/SwimmingWorkout.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  SwimmingWorkout.swift
+//  Workout
+//
+//  Created by Marco Boschi on 04/03/2017.
+//  Copyright © 2017 Marco Boschi. All rights reserved.
+//
+
+import Foundation
+import HealthKit
+import MBLibrary
+
+class SwimmingWorkout: Workout {
+	
+	override init(_ raw: HKWorkout, delegate del: WorkoutDelegate?) {
+		super.init(raw, delegate: del)
+		
+		if #available(iOS 10, *) {
+			self.addDetails()
+			self.addRequest(for: HKQuantityTypeIdentifier.distanceSwimming.getType()!, withUnit: .kilometer(), andTimeType: .ranged, searchingBy: .workout)
```

Comment:
It's indeed very similar as shown below
![img_5010](https://cloud.githubusercontent.com/assets/235227/23766986/a2c87cf4-04e5-11e7-8d96-d60ed560162a.PNG)